InvenTree/templates/InvenTree/settings/user_display.html

```diff
@@ -100,8 +100,11 @@ <h4>{% trans "Language Settings" %}</h4>
     </form>
     </div>
     <div class="col-sm-6">
-        <h4>{% trans "Help the translation efforts!" %}</h4>
-        <p>{% blocktrans with link="https://crowdin.com/project/inventree" %}Native language translation of the web application is <a href="{{link}}">community contributed via crowdin</a>. Contributions are welcomed and encouraged.{% endblocktrans %}</p>
+        <div class='alert alert-block alert-success'>
+            <h5>{% trans "Help the translation efforts!" %}</h5>
+            <p>{% trans "Native language translation of the web application is community contributed via crowdin. Contributions are welcomed and encouraged." %}</p>
+            <p><a href="https://crowdin.com/project/inventree">{% trans "InvenTree Translation Project" %}</a></p>
```

Comment:
@SchrodingersGat suggestion: we removed the project name from all translations a while ago to enable whitelabeling - could we change this to Translation project?